test/lib/heuristics/periodic_test.rb

```diff
@@ -243,10 +243,11 @@ def test_returned_vehicles_ids_with_two_stage_cluster
       problem[:configuration][:preprocessing][:partitions] = TestHelper.vehicle_and_days_partitions
       problem[:configuration][:schedule] = { range_indices: { start: 0, end: 83 } }
 
-      solutions = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:demo] }},
-                                               TestHelper.load_vrp(self, problem: problem), nil)
+      solutions = OptimizerWrapper.wrapper_vrp(
+        'demo', { services: { vrp: [:demo] }}, TestHelper.load_vrp(self, problem: problem), nil
+      ).vrp_result
       assert solutions[0]
-      route_vehicle_ids = solutions[0].routes.collect{ |route| route.vehicle.id }
+      route_vehicle_ids = solutions[0]['routes'].collect{ |route| route['vehicle_id'] }
```

Comment:
Why do we revert from method to string hash? Issue with rebase?

Reply:
Seems to be an artifact from a rebase